fpp

```diff
@@ -25,6 +25,9 @@ PYTHONCMD="python"
 function doProgram {
   # process input from pipe and store as pickled file
   $PYTHONCMD "$BASEDIR/src/processInput.py" "$@"
+  # if it failed, just fail now and exit the script
+  # this works for the looping -ko case as well
+  if [[ $? != 0 ]]; then exit $?; fi
   # now close stdin and choose input...
   exec 0<&-
 
```

Comment:
github comments are lame and wont let me write below, but should we also have the same for `$BASEDIR/src/choose.py`? its less likely to fail but it might be handy for things like when the user quits and doesnt want to run the shell script. let me file a followup for that